src/SignalRServiceExtension/Config/OptionsSetup.cs

```diff
@@ -39,7 +39,7 @@ public OptionsSetup(IConfiguration configuration, ILoggerFactory loggerFactory,
 
         public void Configure(ServiceManagerOptions options)
         {
-            options.ConnectionString = configuration[connectionStringKey];
+            options.ConnectionString = configuration.GetConnectionString(connectionStringKey) ?? configuration[connectionStringKey];
```

Comment:
A common pattern for azure connection string settings

Reply:
Ok.

Reply:
> Cool, thanks for looking into it. Should the attribute just be `protected` and not `protected internal`?

It should be used in `SignalRTriggerBindingProvider` at https://github.com/Azure/azure-functions-signalrservice-extension/pull/263/files/077f73bbfb51e6b42f967170b8a1cabe713b16c3